File: Lab 4/t3.c
#include <stdio.h>

// #define ABS(a) ((a)>=0?(a):(-a))

int main (void)
{
  int a = 0, b = 0;
  puts ("Please input two integers:");
  scanf ("%d %d", &a, &b);
  printf (
    "Between %d and %d, %d has a greater absolute value.\n",
    a, b,
    // ABS(a) >= ABS(b) ? a : b
    (a >= 0 ? a : -a) >= (b >= 0 ? b : -b) ? a : b
  );
  return 0;
}

File: Lab 4/t3.go
package main

import "fmt"

func main() {
  var a, b int = 0, 0
  fmt.Println("Please input two integers:")
  fmt.Scanf("%d %d", &a, &b)
  fmt.Printf("Between %d and %d, %d has a greater absolute value.\n", a, b, bigger(abs(a), abs(b)))
}

func abs(a int) int {
  if a >= 0 {
    return a
  } else {
    return -a
  }
}

func bigger(a, b int) int {
  if a >= b {
    return a
  } else {
    return b
  }
}
